packages/mds-cache/index.ts

```diff
@@ -143,17 +143,17 @@ async function getEventsInBBox(bbox: BoundingBox) {
   return client.georadiusAsync('locations', lng, lat, radius, 'm')
 }
 
-async function hreads(suffixes: string[], device_ids: UUID[]): Promise<CachedItem[]> {
+async function hreads(suffixes: string[], ids: UUID[], prefix = 'device'): Promise<CachedItem[]> {
```

Comment:
Might be a good idea to make `prefix` a string literal that informs usage.
```
prefix: 'device' | 'provider' = 'provider'
```

Reply:
That makes sense, I'm also a fan of leaning on the type system to write more correct code.